app/assets/images/payment_icons/kasikorn.svg

```diff
@@ -0,0 +1,2 @@
+    <title id="pi-kasikorn">Kasikorn</title><path opacity=".1" d="M35 0H3a3 3 0 0 0-3 3v18a3 3 0 0 0 3 3h32a3 3 0 0 0 3-3V3a3 3 0 0 0-3-3z"/><path fill="#FFF" d="M35 1a2 2 0 0 1 2 2v18a2 2 0 0 1-2 2H3a2 2 0 0 1-2-2V3c0-1.1.9-2 2-2h32"/><image overflow="visible" width="2126" height="2104" xlink:href="../../../../../../../../Downloads/BBL-BS-E.jpg" transform="matrix(.24 0 0 .24 -8050.3 -8138.5)"/><path fill="#E40326" d="M19 1.4A10.6 10.6 0 1 1 8.4 12C8.4 6.3 13.1 1.5 19 1.5m4.8 18c.2 0 .2-.1.3-.2l.2-.2.2-.1c.1 0 .2 0 .2-.2 0 0 .2 0 .2-.2.1 0 .3 0 .3-.2l.1-.2c.1 0 .2 0 .3-.2l.2-.3.2-.1A8.8 8.8 0 0 0 24.4 5a8.8 8.8 0 0 0-13 11.7l.8 1 .2.1c0 .2 0 .2.2.3l.2.2.2.2.3.2.2.1.2.2.1.2c.2 0 .3.1.3.2h.2l.3.2c0 .2.2.2.3.2a8.4 8.4 0 0 0 6.1.8 10 10 0 0 0 1.6-.5c.3-.1.6-.2.8-.5l.3-.1h.1"/><path fill="#199C43" d="M12.2 17.6a6 6 0 0 0 2-.6 3.3 3.3 0 0 1 3 0l1.3.6h.8c.7-.1 1.2-.4 1.8-.7.4-.2.9-.2 1.3-.2.7 0 1.3.3 1.9.6.4.2.8.2 1.3.3h.2l-.2.3h-.1c-.7 0-1.3-.4-1.9-.7-.6-.3-1.2-.4-1.9-.2-.5 0-1 .4-1.4.6l-1 .2c-.8 0-1.5-.1-2.1-.5a4 4 0 0 0-1.4-.4c-.6 0-1.2.2-1.7.4-.6.3-1.1.5-1.7.5l-.2-.2"/><path fill="#199C43" d="M25.3 18l-.1.3-.9-.2-.6-.4-1.2-.3c-.6 0-1.1.2-1.7.4a4 4 0 0 1-2.4.5l-1.3-.5c-.5-.2-1-.4-1.4-.4a3 3 0 0 0-1.3.3l-1.6.6-.2-.2 1-.3 1.2-.5c.7-.2 1.2-.2 1.8 0l1.4.6c.6.2 1.2.2 1.9 0l1.5-.6c.5-.2 1.1-.2 1.7 0 .4 0 .7.2 1 .4l1.2.4"/><path fill="#199C43" d="M25 18.5l-.3.2-.8-.3c-.4-.3-.9-.5-1.4-.5-.8 0-1.4.3-2 .6l-1 .3-1.5-.1-1.3-.6c-.4-.1-.9-.3-1.4-.2-.7.1-1.4.5-2 .8l-.3-.2 1.3-.5a3 3 0 0 1 2.3-.2l1 .5c.4.1.9.3 1.4.3.6 0 1.1-.2 1.6-.5.5-.2 1-.4 1.5-.4s1 0 1.5.2c.4.2.8.5 1.3.6"/><path fill="#199C43" d="M24.5 18.9l-.2.1-1.4-.5c-.5-.2-1-.1-1.4 0l-1.5.7c-.5.1-1.1.2-1.6 0-.6 0-1-.3-1.5-.5-1.2-.6-2.2-.3-3.2.3a.4.4 0 0 1-.2-.2l1.4-.5c.4-.2 1-.2 1.4 0 .5 0 1 .3 1.5.5.3.2.7.3 1 .3a4 4 0 0 0 2-.5l1-.4h1.4l1.3.7"/><path fill="#199C43" d="M24 19.2s0 .2-.2.2v-.1l-.5-.2a2 2 0 0 0-1.4-.2c-.6.1-1 .4-1.6.7-.5.2-1 .2-1.4.2-.7 0-1.3-.3-2-.6-.3-.2-.8-.3-1.2-.3-.4 0-.8 0-1.1.2-.2 0-.4 0-.4.3l-.4-.2.5-.3c.4-.2.8-.2 1.2-.2.8 0 1.4.3 2.1.6.4.2.9.3 1.3.3.7 0 1.4-.3 2-.5.5-.3 1.1-.5 1.8-.4.5 0 1 .2 1.4.5"/><path fill="#199C43" d="M23.4 19.6c-.5-.2-1-.3-1.6-.1-.4 0-.8.3-1.2.4-.7.4-1.4.5-2 .4-.5 0-1-.2-1.4-.4-.7-.4-1.4-.6-2.1-.4l-.5.1c-.1 0-.2 0-.3-.2l.5-.1c.5-.2 1-.1 1.5 0 .6 0 1 .4 1.6.6l1 .2c.7 0 1.3-.2 1.8-.5.6-.3 1.2-.5 2-.4l1 .3-.3.1M23.8 19.4zM18.8 13.7v-3c0-.8.3-1.5 1-2l1.4-1 2-1.3.6-.9h.1v3.1c0 .5-.2.8-.6 1.1-.5.5-1.2.9-1.8 1.3l-1.7 1.2c-.4.3-.6.7-.6 1.2V17c0 .1 0 .2-.2.2-.1 0-.2 0-.2-.2v-2.5-.8zM18.4 13.7V17l-.1.2c-.2 0-.3 0-.3-.2v-2.6-1.2c-.2-.5-.4-1-.8-1.3l-1.9-1.4-.9-.9c-.3-.4-.5-.8-.5-1.3V5.6v-.2l.2.1c.2.5.5.9 1 1.2A71.9 71.9 0 0 0 17.7 9c.5.4.6 1 .6 1.5v3.2z"/><path fill="#199C43" d="M23.8 9.5c.2 0 .1.2.1.2V12c0 .6-.4 1.1-1 1.5l-1.8 1.2a3 3 0 0 0-1 1.2l-.1.6v.5c0 .1-.1.2-.3.2l-.1-.2v-3.3c0-.7.3-1.2.8-1.6l2.5-1.6.8-.6c0-.1.2-.2.1-.4"/></svg>
```

Comment:
Hello @ainzin, this icon contains `<image>` which does not follow our [guideline](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#markup)